iceoryx_posh/include/iceoryx_posh/capro/service_description.hpp

```diff
@@ -134,10 +130,9 @@ class ServiceDescription
     /// @brief Returns the scope of a ServiceDescription
     Scope getScope() noexcept;
 
-    /// @todo remove any*string from doxygen?
-    ///@brief Returns true for valid ServiceDescription
-    /// false for ServiceDescription that contains either of InvalidID/InvalidIDString  AnyService/AnyServiceString,
-    /// AnyInstance/AnyInstanceString, AnyEvent/AnyEventString.
+    /// @brief Returns true for valid ServiceDescription
+    /// false for ServiceDescription that contain InvalidStrings.
+    /// @return bool, true if ServiceDescription is valid, false otherwise
```

Comment:
I see no reason to mention the return type in the brief description.

Reply:
It doesn't hurt either ;)